core/src/test/java/com/google/errorprone/bugpatterns/IdentityConversionTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2022 The Error Prone Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.errorprone.bugpatterns;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for {@link IdentityConversion}. */
+@RunWith(JUnit4.class)
+public class IdentityConversionTest {
+
+  private final BugCheckerRefactoringTestHelper helper =
+      BugCheckerRefactoringTestHelper.newInstance(IdentityConversion.class, getClass());
+
+  @Test
+  public void removeValueOf() {
```

Comment:
Doesn't seem like we need to have two tests for the different kind of conversion methods. (But if we add a second suggested fix we should test that.)